chemprop/data/data.py

```diff
@@ -133,6 +150,30 @@ def set_features(self, features: np.ndarray) -> None:
         """
         self.features = features
 
+    def set_atom_descriptors(self, atom_descriptors: np.ndarray) -> None:
+        """
+        Sets the atom descriptors of the molecule.
+
+        :param atom_descriptors: A 1D numpy array of features for the molecule.
+        """
+        self.atom_descriptors = atom_descriptors
+
+    def set_atom_features(self, atom_features: np.ndarray) -> None:
+        """
+        Sets the atom features of the molecule.
+
+        :param atom_features: A 1D numpy array of features for the molecule.
+        """
+        self.atom_features = atom_features
+
+    def set_bond_descriptors(self, bond_descriptors: np.ndarray) -> None:
+        """
+        Sets the bond descriptors of the molecule.
+
+        :param bond_descriptors: A 1D numpy array of features for the molecule.
+        """
+        self.bond_descriptors = bond_descriptors
```

Comment:
I think the nomenclature here is quite misleading: Running with the option `--atom_descriptors feature` triggers the features to be appended to the regular atom/bond features (or overwritten) before message passing, but this is now described by `atom/bond_descriptors`. I think we should thus name change `features` to `descriptors` and vice versa in all of the function and variable names.

Reply:
Solved!